lib/catch-links.js

```diff
@@ -18,6 +23,14 @@ module.exports = function (root, cb) {
     if (href) {
       var isUrl
 
+      // Check for magnet link
+      if(magnet_pattern.test(href))
+      {
+        // throw to OS (safe?)
+        shell.openExternal(href)
+        return
+      }
+
       try {
         var url = new URL(href)
```

Comment:
Whoops, github nuked my other review note.

This is already parsing magnet urls, but they are getting routed as 'internal' urls. To change this, you can update the check to something like this:

```js
if (isUrl && (url.host || url.protocol === 'magnet:')) {
```

Or if you want to keep the regexp check:

```js
if (isUrl && (url.host || (url.protocol === 'magnet:' && magnet_pattern.test(href)))) {
```

Reply:
oh ya that makes sense! pushed a simpler version